components/header/_header2.scss

```diff
@@ -0,0 +1,65 @@
+$header2-bp: 36em;
+
+.header2 {
+  display: block;
+  overflow: hidden;
+  padding: 1.4rem 0 1rem;
+  &::before {
+    content: "";
+    display: block;
+    width: 100%;
+    height: 8px;
```

Comment:
Should this be in `rems`?

Reply:
Are you referring to just the height? Should be a straight forward fix. Not sure that rems can't directly substitute percentages.